.github/workflows/custom_check.yml

```diff
@@ -12,10 +12,10 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: Set up Python 3.7
+    - name: Set up Python 3.10
       uses: actions/setup-python@v4
       with:
-        python-version: 3.7
+        python-version: 3.10
```

Comment:
I didn't update our release package Python versions -- we should bump after 24.1 is out the door though.  For non-release workflows I bumped to Python 3.10 just cause.